website/content/sdk/installation.md

```diff
@@ -4,182 +4,87 @@ sidebar_position: 1
-
-Go [here](https://docs.conda.io/en/latest/miniconda.html#latest-miniconda-installer-links) to find the download for your operating system or use the links below:
-
-- Apple-Silicon Systems: [Miniconda for MacOS](https://repo.anaconda.com/miniconda/Miniconda3-latest-MacOSX-arm64.pkg)
-- Intel-based Mac Systems: [Miniconda for MacOS](https://repo.anaconda.com/miniconda/Miniconda3-latest-MacOSX-x86_64.sh)
-- Linux and WSL Systems: [Miniconda for Linux](https://repo.anaconda.com/miniconda/Miniconda3-latest-Linux-x86_64.sh)
-- Raspberry PI Systems: [Miniconda for Raspberry PI](https://repo.anaconda.com/miniconda/Miniconda3-latest-Linux-aarch64.sh)
-- Windows Systems: [Miniconda for Windows](https://repo.anaconda.com/miniconda/Miniconda3-latest-Windows-x86_64.exe)
-
-
-**NOTE for Apple Silicon Users:** Install Rosetta from the command line: `softwareupdate --install-rosetta`
-
-**NOTE for Windows users:** Install/update Microsoft C++ Build Tools from [here](https://visualstudio.microsoft.com/visual-cpp-build-tools/).
-</details>
-
-<details><summary>CMake (Mac and Linux only)</summary>
-
-If you have a **MacBook**, check if homebrew is installed by running `brew --version`
-
-If Homebrew is not installed, run:
-
-```bash
-/bin/bash -c "$(curl -fsSL https://raw.githubusercontent.com/Homebrew/install/HEAD/install.sh)"
-brew install cmake
-brew install gcc
+```shell
+pip install openbb
 ```
 
-If Homebrew is already installed:
+This method provides access to the data aggregation and charting functions of the OpenBB SDK. It does not provide access to the advanced features that are provided by the Portfolio Optimization and Machine Learning toolkits.
 
-```bash
-brew install cmake
-brew install gcc
-```
+If need to use the SDK with the Machine Learning and Portfolio Optimization toolkits we recommend following the [Full Installation instructions](#full-installation).
 
-If you have a **Linux** computer, use the following script:
+## Full Installation
 
-```bash
-sudo apt update && sudo apt upgrade
-sudo apt install -y gcc cmake
-```
-</details>
+To access the full functionality of the SDK with all the default toolkits, we recommend installing the SDK from source code [(link)](/terminal/installation/source). This will enable you to use the SDK with all the toolkits and contribute to the OpenBB project.
 
-## Create Python Environment
+However, if you already have [the requirements listed at the beginning of the source code installation instructions](/terminal/installation/source) installed on your system, you can alternatively install the SDK toolkits from PyPI.
 
-The first step is to create the virtual Python environment. When a terminal window is opened, if the base Conda environment - look for `(base)` to the left of the cursor on the command line - is not activated automatically, find the path for it by entering:
+Install both ML and Portfolio Optimization toolkits with:
 
-```console
-conda env list
+```shell
+pip install openbb[all]
```

Comment:
Not having quotes ("openbb[all]") is going to fail on some systems (including mine).